lib/player.js

```diff
@@ -4703,6 +4703,17 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
            false;
   }
 
+  /**
+   * Get if the player is playing live or in-progress content. If the player
+   * has not loaded content, this will return <code>false</code>.
+   *
+   * @return {boolean}
+   * @export
+   */
+  isDynamic() {
+    return this.isLive() || this.isInProgress();
+  }
```

Comment:
new API, so its a feature, not a bugfix.

Reply:
Discussed via chat, and settled on "fix".  I'll remove the `@export` for now.